sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2460,6 +2603,7 @@ public partial class GremlinGraphPropertiesConfig : Azure.ResourceManager.Cosmos
     public partial class GremlinGraphResourceInfo
     {
         public GremlinGraphResourceInfo(string id) { }
+        public long? AnalyticalStorageTtl { get { throw null; } set { } }
```

Comment:
This property is not ready on service side: 
`Azure.RequestFailedException : Message: {"code":"BadRequest","message":"Message: {\"Errors\":[\"The input content is invalid - 'analyticalStorageTtl' is not a valid property in the current payload.\"]}`